platform/windows/export/export_plugin.cpp

```diff
@@ -242,16 +243,22 @@ Error EditorExportPlatformWindows::export_project(const Ref<EditorExportPreset>
 	if (embedded) {
 		pck_path = pck_path.get_basename() + ".tmp";
 	}
+
 	Error err = EditorExportPlatformPC::export_project(p_preset, p_debug, pck_path, p_flags);
-	if (p_preset->get("codesign/enable") && err == OK) {
+	if (err != OK) {
+		// Message is supplied by the subroutine method.
+		return err;
+	}
```

Comment:
Don't see a point in trying to export ZIP if we failed here already. Don't do it on Linux either.